### PR TITLE
Update readme.md

### LINKED ISSUE #48: ReadMe
Create ReadMe for run first time your project

### PR DESCRIPTION
**Pull request by Morel-Victor**

### CODE REVIEW

**Review by LuigiMenez — DISMISSED** [review]

File: README.md [hunk 1]
```diff
@@ -1,13 +1,22 @@
...
-## First steps
+## Description ✍🏻
 
-### Install these extensions
+Shaker is a search engine of cocktails.
 
-- [ESLint](https://marketplace.visualstudio.com/items?itemName=dbaeumer.vscode-eslint)
-- [Prettier](https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode)
-- [StyleLint](https://marketplace.visualstudio.com/items?itemName=stylelint.vscode-stylelint)
+He references more than 700 cocktails.
```
Comment (LoicBrassart) [inline README.md]: "It" :p

File: README.md [hunk 2]
```diff
@@ -1,13 +1,22 @@
...
 
-### Install `husky`, which will check your code everytime you commit files
+Each one has his recipe to make the best cocktails.
```
Comment (LoicBrassart) [inline README.md]: it's recipe

**Review by LoicBrassart — CHANGES_REQUESTED** [review]
Minor typos

File: README.md (hunk 1 above)
Comment (PierreGoe) [reply README.md]: solve

File: README.md (hunk 2 above)
Comment (PierreGoe) [reply README.md]: solve

**Review by LuigiMenez — APPROVED** [review]
Good.

**Review by LoicBrassart — APPROVED** [review]
Better ;-)

**Pull request merged**